pkg/controller/controller.go

```diff
@@ -395,6 +398,12 @@ func (p *csiProvisioner) ProvisionExt(options controller.ProvisionOptions) (*v1.
 		}
 	}
 
+	if p.enableNodeCheck && options.SelectedNode.Name != os.Getenv("NODE_NAME") {
+		return nil, controller.ProvisioningNoChange, &controller.IgnoredError{
```

Comment:
I think we should return ProvisioningFinished here, so that lib-external-provisioner will stop retrying to call Provision().  cc @jsafrane to confirm

Hmmm although will that cause lib-external-provisioner to remove "selected-node" annotation to make scheduler retry?  Maybe we need lib-external-provisioner to be aware of this behavior too...